packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -234,7 +234,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 
         // Reference to this client supporting get_ops flow.
         connectMessage.supportedFeatures = { };
-        if (mc.config.getBoolean("Fluid.Driver.Odsp.GetOpsEnabled") === false) {
+        if (mc.config.getBoolean("Fluid.Driver.Odsp.GetOpsEnabled") !== false) {
```

Comment:
Just checking - `getBooloean` will return `undefined` if the host doesn't pass this setting right? It only returns `false` if the host explicitly sets it to `false`?

Reply:
exactly. undefined means not set